properties/props.sail

```diff
@@ -31,22 +31,6 @@ function encodeDecode(c : Capability) -> Capability =  capBitsToCapability(c.tag
  */
 function capEncodable(c : Capability) -> bool = c == encodeDecode(c)
 
-// $property
-/*
- * THIS is not actually an invariant but is useful for characterising unusable
- * encodings. Ideally we'd like an encoding that ensures base is less than or
- * equal to top, but sadly this isn't true. Counterexamples are unusable
- * encodings. This can happen when a_top ends up being zero and T < B because
- * the attempted corrections can underflow a_top. There are other unusable
- * encodings too, for example those with top larger than 2**32.
- */
-function prop_base_lteq_top(b : CapBits) -> bool = {
```

Comment:
The latex build is failing because you removed this function, which is included in the archdoc in an appendix.

Reply:
The actual problem is the removal of `cap_mantissa_width` as this is referred to in `archdoc/chap-isaref-riscv.tex` in the constant definitions section.

Reply:
Fixed